### PR TITLE
task 1 node js basic

### PR DESCRIPTION
**Pull request by VBugaev**

### CODE REVIEW

File: task_1_NodeJS-basic/task1.2/streamManager.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const csv = require('csvtojson');
+const csvFilePath = `${__dirname}/csv/t1_2.csv`;
```
Comment (Vadim2805) [inline task_1_NodeJS-basic/task1.2/streamManager.js]: Extract hardcoded file paths to the main.js. Implement getting them as arguments. Module should be reusable.

File: task_1_NodeJS-basic/task1.2/streamManager.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+const createCsvInstance = () => csv(parsingOptions)
+    .on('error', console.error)
+    .preFileLine(setHeaderInLowerCase);
+
+module.exports = {
+    doParsingPipeline: () => {
+        const readStream = fs.createReadStream(csvFilePath);
+        const writeStream = fs.createWriteStream(outputFilePath);
+        const pipeline = readStream.pipe(createCsvInstance()).pipe(writeStream);
```
Comment (Vadim2805) [inline task_1_NodeJS-basic/task1.2/streamManager.js]: File should be read line by line. Please check it and rewrite if needed.

Comment (VBugaev) [reply task_1_NodeJS-basic/task1.2/streamManager.js]: done

File: task_1_NodeJS-basic/task1.2/streamManager.js (hunk 1 above)
Comment (VBugaev) [reply task_1_NodeJS-basic/task1.2/streamManager.js]: done

**Pull request merged**